src/log.js

```diff
@@ -64,6 +64,27 @@ export function setReportError(fn) {
 }
 
 /**
+ * An extensible cross-browser error class that correctly supports messages,
+ * stack traces, and instanceof for subclasses.
+ * @abstract
+ */
+export class AbstractError extends Error {
+  /**
+   * @param {string} message
+   */
+  constructor(message) {
+    super(message);
```

Comment:
I ran the unit test for this on our SauceLabs suite and it passed on all platforms. Does that mitigate the danger? Do those tests cover every browser/device we support?